### PR TITLE
feat: implement quote detail page for author

### PR DESCRIPTION
**Pull request by Tedla2001**

### CODE REVIEW

File: authors/detail.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    $i=0;
+    while($line=fgets($fh)) {
+    $line_array = explode(";",$line);
+    if ($index==trim($line_array[0])) {
+        echo "\"".$line_array[1]."\"<br>";
+    }
+    $i = $i + 1;
+    }
+    fclose($fh);
+    }
```
Comment (theogrammer) [inline authors/detail.php]: Do you think this could benefit from a function from our `csv_util.php` class?

File: authors/detail.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<?php
+include '../csv_util.php';
+$line_counter = 0;
+$index = $_GET['index'];
+$authors_array = read_csv('../data/authors.csv');
+$author_name = $authors_array[$index];
+
+function printQuote($filename,$index){
```
Comment (theogrammer) [inline authors/detail.php:8]: `$filename` is not used in the function

**Review by theogrammer — APPROVED** [review]

File: authors/detail.php (hunk 2 above)
Comment (Tedla2001) [reply authors/detail.php:8]: should be fix with the most recent commit

File: authors/detail.php (hunk 1 above)
Comment (Tedla2001) [reply authors/detail.php]: function on csv_util will change the implementation of this function

**Pull request merged**